### PR TITLE
Reconstruct scripts & food detection folder

### PR DESCRIPTION
**Pull request by naili-xing**

### CODE REVIEW

File: examples/models/image_object_detection/food_detection/food101.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from singa_auto.darknet.food_objection_base_model import FoodDetectionBase
+from examples.models.image_object_detection.food_detection.food_objection_base_model import FoodDetectionBase
```
Comment (nudles) [inline examples/models/image_object_detection/food_detection/food101.py:1]: do you need the full path to import the module if all files are in the same folder?

File: scripts/.base_env.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+# Core secrets for SINGA-auto - change these in production!
```
Comment (nudles) [inline scripts/.base_env.sh:21]: better add comments on which variable needs updating when the system is deployed.

File: examples/models/image_object_detection/food_detection/food101.py (hunk 1 above)
Comment (naili-xing) [reply examples/models/image_object_detection/food_detection/food101.py:1]: This is just an example code, If we deploy this app to the singa-auto, we may need to use another way.

File: scripts/.base_env.sh (hunk 2 above)
Comment (naili-xing) [reply scripts/.base_env.sh:21]: ok

**Comment by naili-xing** [comment]
> there are still many scripts..
> if script A and B are executed together, then merge them into a single script.
> 
> can you delete some dockerfiles?
> If a dockerfile has nothing except simple pip install or file copy, then just run these commands when launching the docker container. in this way, we do not need to maintain many docker images.

yes, we can delete some dockerfiles

**Pull request merged**